pkg/square/square_benchmark_test.go

```diff
@@ -1,56 +1,44 @@
 package square_test
 
-import (
-	"fmt"
-	"testing"
+// func BenchmarkSquareConstruct(b *testing.B) {
```

Comment:
[blocking] is this intended to be committed commented out? Instead of merging commented code can we:

delete if no longer relevant
uncomment and fix
worst case: delete with a GH issue to re-add?